src/sample_weight_audit/estimator_check.py

```diff
@@ -309,6 +312,20 @@ def multifit_over_weighted_and_repeated(
             est_weighted = clone(est).set_params(**extra_params_weighted)
             est_repeated = clone(est).set_params(**extra_params_repeated)
 
+        if shuffle:
+            rng = np.random.default_rng()
```

Comment:
For debuggability, we should make the code reproducible:

```suggestion
            rng = np.random.default_rng(seed)
```